### PR TITLE
update libg with find and replace

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

update libG to latest:
change log:
* reduced memory leaks in solid loft workflow
* reduced memory leaks in polycurve workflows
* reduced memory leaks when throwing exceptions
* swig 4.0
* bug fixes in point projection and point prune duplicates
* faster pt/pt distance calculations
* updated docs for `projectInputOnto` nodes
* NurbsSurfaces operations should return exact surface types in more cases

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -83,8 +83,8 @@ limitations under the License.
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProtoGeometry, Version=2.9.0.3837, Culture=neutral, processorArchitecture=AMD64">
-      <HintPath>..\packages\DynamoVisualProgramming.LibG_225_0_0.2.9.0.3837\lib\net48\ProtoGeometry.dll</HintPath>
+    <Reference Include="ProtoGeometry, Version=2.11.0.4156, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\DynamoVisualProgramming.LibG_225_0_0.2.11.0.4156\lib\net48\ProtoGeometry.dll</HintPath>
```
Comment (aparajit-pratap) [inline src/DynamoCore/DynamoCore.csproj:87]: Not that it matters much but we try to match this version with that for ASM227 (or whichever is the current latest).

Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:87]: Sorry @aparajit-pratap I'm not sure I get it - LibG227 nuget had a different version number, and was pointing to a newer commit, I wanted all nugets to contain the same code... 

atleast I think thats what I did :) - grabbed everything on nuget from yesterday - 227 had an extra commit I believe.

Comment (aparajit-pratap) [reply src/DynamoCore/DynamoCore.csproj:87]: What I'm saying is that we should reference the protogeometry version that's coming from ASM227 here like we're doing in all other projects.

Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:87]: ahh, okay, will make the change.

**Comment by mjkkirschner** [comment]
@aparajit-pratap - I'm rerunning the tests, not sure why these 2 tests failed...

**Comment by mjkkirschner** [comment]
@mjkkirschner these tests fail locally, as well, very strange, not sure how modifying libG could effect them - will investigate.

**Comment by aparajit-pratap** [comment]
I guess we shouldn't wait that long before pulling libg nugets into Dynamo. Especially in this case, where we have some big changes in libg, it could be hard to track down which libg change is actually causing the regression.

**Review by aparajit-pratap — APPROVED** [review]

**Comment by mjkkirschner** [comment]
this passed here:
https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/2095/

I'll merge after the final test job completes.

**Pull request merged**